src/edu/washington/escience/myria/util/DeploymentUtils.java

```diff
@@ -335,24 +349,30 @@ public static void mkdir(final String address, final String remotePath) {
    * @param remotePath e.g. /tmp/test
    */
   public static void rsyncFileToRemote(final String localPath, final String address, final String remotePath) {
-    rsyncFileToRemote(localPath, address, remotePath, false);
+    rsyncFileToRemote(localPath, address, remotePath, null, null);
```

Comment:
I realized that this method is never used, so removed. I also added `@Nonnull` to `includes` and `excludes`.